### DISCUSSION TITLE
New release?

### DISCUSSION THREAD

**Discussion by 0-wiz-0** [gollum/gollum#2026] [General]
Hi!

I reported a bug recently and was told that this was fixed in the current development version, and that a new release would be coming - what are the plans for that?

(I'd like to package this for pkgsrc and  I see that the current release and the current development version have wildly different version requirements for some dependencies in their gemspec files. So if a new release is coming soon, packaging only the newer versions would save time.)

Thanks,
Thomas

**Comment by dometto** [comment]
Hi @0-wiz-0! We should definitely release soon. We're all volunteers and the second half of 2023 was a tough one for working on gollum. I think what happened is that we postponed so much that we started to see issues for which we really wanted to get fixes into 6.0, but then didn't really have time to address them... Perhaps a good new year's resolution is to come up with stricter and better defined release timelines. :)

But I would definitely recommend just focusing on the dependencies for the newer version -- and thanks for packaging gollum for pkgsrc!

**Comment by dometto** [comment]
To (ab)use this discussion for a bit of planning:

The last issue that was officially in the 6.0 milestone is addressed in https://github.com/gollum/gollum/pull/2022. @bartkamphorst @benjaminwil if you could help me take some decisions about removing the command line switches, that would be really helpful, I remain very unsure whether it makes sense to go through with that.

@bartkamphorst and I were also wondering whether it might be possible to get at least rudimentary [dark mode support](https://github.com/gollum/gollum/pull/1938) into 6.0. I think we should either make a concrete plan to do that in the very short term, or just forget about it and release that as part of 6.1. :)

**Reply by benjaminwil** [reply]
I would vote pushing dark mode to 6.1 if it's there's any risk it would delay the release further. The upside to pushing this to the 6.1 release is that it makes that release very shiny ✨
